web/src/components/speaker/Links.vue

```diff
@@ -1,22 +1,46 @@
 <template>
   <div class="mt-3">
-    <social-button :link="twitter">
-      <i class="fab fa-twitter" />
-    </social-button>
-    <social-button :link="linkedIn">
-      <i class="fab fa-linkedin-in" />
-    </social-button>
-    <social-button :link="facebook">
-      <i class="fab fa-facebook-f" />
-    </social-button>
-    <social-button :link="website">
+    <v-tooltip top :disabled="$vuetify.breakpoint.width < 600">
```

Comment:
I meant to comment this the other day but it seems I didn't actually submit it 😓 . Can we change this to a visible tooltip on narrow, but just have it on the bottom? Or leave it overlapping? The reason is sometimes desktop uses make their browser window different sizes. We don't have to worry about hiding it on mobile because there is no ability to hover on mobile.

Reply:
I changed the tooltip to bottom, because the code will double in size if I use v-if or v-show to change the position depending on the screen size. Just realized that it only overlaps in tablet mode, because the social icons stay to the bottom left for desktop and mobile. Sorry that my screenshot above was actually for a tablet screen size!